packages/gazelle_cli/lib/commands/delete/delete_project.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:io';
+
+/// Uninstalls a Gazelle project.
+class DeletingProjectError {
+  /// The error message.
+  final String message;
+
+  /// Creates a [DeletingProjectError].
+  const DeletingProjectError() : message = "Failed to delete project!";
+}
+
+/// Asks for confirmation before deleting a project.
+///
+/// Returns `true` if user confirms, `false` otherwise.
+Future<bool> getConfirmation(String message) async {
+  print('$message (y/N)');
+  String? answer = stdin.readLineSync();
+  if (answer?.toLowerCase() == 'y' || answer?.toLowerCase() == 'yes') {
+    return true;
+  }
+  return false;
+}
```

Comment:
If you're using `stdin.readLineSync()` I would suggest to make the function synchronous.
So the function signature would be `bool getConfirmation(String message)`.

We could move this inside the `lib/commons/functions` directory, so we can use it also for other commands!
What do you think?

Reply:
I think it's a great idea and implemented the same created a file and added the following code to it.